File: AMS.API/DTO/BatchMaster/BatchMasterResponse.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace AMS.API.DTO.BatchMaster
{
    public class BatchMasterResponse
    {
        #region Constructor

        public BatchMasterResponse()
        {
            BatchMaster = new List<BatchMaster>();
            OperationStatus = new OperationStatus();
        }
        #endregion Constructor

        #region Class Properties

        public List<BatchMaster> BatchMaster { get; set; }

        public OperationStatus OperationStatus { get; set; }

        #endregion
    }
}

File: AMS.Web/src/app/models/batch-master/batch-master-response.ts
import { BatchMaster } from "./batch-master";
import { OperationStatus } from "../shared/operation-status";

export class BatchMasterResponse 
{
    //#region Constructor

    public BatchMasterResponse()
    {
        this.BatchMaster = new Array<BatchMaster>();
        this.OperationStatus = new OperationStatus();
    }

    //#endregion Constructor

    //#region Class Properties

    BatchMaster:Array<BatchMaster>;

    OperationStatus:OperationStatus;

    //#endregion
}
